### ISSUE TITLE
Multi-choice field no longer has search bar in Chrome

### ISSUE THREAD

**Issue by dboyer** [OpenTreeMap/otm-core#2751]
See attached screenshot. The search bar to type is still visible in Firefox and IE.
![otm_multichoice_select](https://cloud.githubusercontent.com/assets/516764/18133556/17768c98-6f69-11e6-98e8-cadd82564f8e.png)

**Comment by dboyer** [comment]
I'm seeing this intermittently and would like to see if someone else can reproduce it.

**Comment by jwalgran** [comment]
1. If one does not already exist, create a multi-choice UDF with 3 choices and make it writeable by all users
1. Add a tree from the map page and use the "continue editing" checkbox to navigate to the edit page
1. Verify that the multichoice dropdown has a search field
1. Click save then click edit
1. The multi-choice dropdown no longer has a search field.

Based on these finding I suspect that the updated save behavior (#2720), may have caused the regression.

**Comment by RickMohr** [comment]
@dboyer, I don't understand what you mean by "search bar".
@jwalgran, I don't understand what you mean by "search field".

Are we talking about:
* the dropdown that appears when you click on a multichoice field in Edit mode?
* the advanced search field that lets you search for the multichoice field?

For me both of the above work fine through Edit / Save / Edit. Is it something else?

**Comment by dboyer** [comment]
I used "search bar" to refer to the bar with a magnifying glass that appears in the dropdown above the list of multichoice options when you are in Edit mode on a detail page. The first comment on this thread shows a screenshot of what I saw in Chrome when testing yesterday. All the choices were there but no search bar to start typing text which limited the list of options.

I did not have a problem with using the advanced search to search for a multichoice field.
